File: tests/Fortran/Fixed/0005.f
       program test
       integer, parameter :: N = 10
       real v1(N), v2(N), v3(N)

!$acc update device(v1)

!$acc update device(v1)
!$acc& device(v2,v3)

!$acc update device(v1)
!$acc1 device(v2)
!$acc2 device(v3)
       end program

File: tests/C/0005.c
int main (int argc, char *argv[])
{
	int v1[1000], v2[1000], v3[1000];

	#pragma acc update device(v1)

	#pragma acc update device(v1) \
	device(v2,v3)

	#pragma acc update device(v1) \
	  device(v2) \
	  device(v3)

	return 0;
}
